components/Article/Page.js

```diff
@@ -250,6 +250,10 @@ class ArticlePage extends Component {
       />
     ) : null
 
+    const formatColor =
```

Comment:
Something like this would be more explicit and avoid `article.content.meta` (an internal, you should add color to the main meta query):
```js
const formatMeta = meta && (
  meta.template === 'format'
    ? meta
    : meta.format && meta.format.meta
)
const formatColor = formatMeta && formatMeta.color
```